tests/unit/test_session_converter.py

```diff
@@ -8,8 +8,22 @@
 import nwb2bids
 
 
-def test_session_converter_initialization(minimal_nwbfile_path: pathlib.Path, temporary_bids_directory: pathlib.Path):
-    session_converters = nwb2bids.SessionConverter.from_nwb_directory(nwb_directory=minimal_nwbfile_path.parent)
+def test_session_converter_directory_initialization(
+    minimal_nwbfile_path: pathlib.Path, temporary_bids_directory: pathlib.Path
+):
+    nwb_paths = [minimal_nwbfile_path.parent]
+    session_converters = nwb2bids.SessionConverter.from_nwb_paths(nwb_paths=nwb_paths)
+
+    assert isinstance(session_converters, list)
+    assert len(session_converters) == 1
+    assert isinstance(session_converters[0], nwb2bids.SessionConverter)
+
+
+def test_session_converter_file_path_initialization(
+    minimal_nwbfile_path: pathlib.Path, temporary_bids_directory: pathlib.Path
```

Comment:
Add a test to test `nwb_paths` being a mix of file and directory paths?

Reply:
Done